Eduroam source more flexible (#7220)
inverse-inc/packetfence: docs/installation/authentication_mechanisms.asciidoc

Open the PacketFence administration web interface and go to _Configuration -> Policies and Access Control -> Authentication Sources_.

> First create as much RADIUS sources as you have Eduroam servers to define.

Reviewer: ```suggestion
First create RADIUS sources for each Eduroam servers you want to define.
```
